webpack.config.js

```diff
@@ -217,6 +217,12 @@ module.exports = [
         },
         performance: getPerformanceHints(5 * 1024)
     }),
+    Object.assign({}, config, {
```

Comment:
I'd say Matomo is relevant enough to warrant its inclusion by default, so no need to create a separate bundle. See how we do for Amplitude.